python-sdk/tutorials/nuscenes_lidarseg_tutorial.ipynb

```diff
@@ -62,6 +62,18 @@
     "As you can see, you do not need any extra libraries to use nuScenes-lidarseg. The original nuScenes devkit which you are familiar with has been extended so that you can use it seamlessly with nuScenes-lidarseg."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that if you are using MacOSX and you [face issues with rendering](https://github.com/matplotlib/matplotlib/issues/13414), try adding one of the following to your `.matplotlib/matplotlibrc` file:\n",
+    "```\n",
+    "backend : TKAgg\n",
+    "## or ##\n",
+    "backend : macOSX\n",
+    "```"
```

Comment:
Let's put these in https://github.com/nutonomy/nuscenes-devkit/blob/nuscenes_v2.0/docs/installation.md , not in the tutorial.